python_modules/dagster/dagster/_core/definitions/decorators/sensor_decorator.py

```diff
@@ -23,9 +23,10 @@
     SensorDefinition,
     SkipReason,
 )
-from ..target import ExecutableDefinition
+from ..target import CoercibleToAutomationTarget, ExecutableDefinition
```

Comment:
I don't think `CoercibleToAutomationTarget` should be part of our public API surface area.

Reply:
hmm... you think we just inline a long `Union` for the `target` param?

Reply:
Yeah just inline the union for now